src/design.cpp

```diff
@@ -2861,9 +2855,6 @@ static void desCreateDefaultTemplate()
 /* Remove the design widgets from the widget screen */
 void intRemoveDesign()
 {
-	//save the current design on exit if it is valid
-	saveTemplate();
```

Comment:
This seems to be unnecessary, cause every time anything changes in the design it gets immediately saved.